ui-kit/src/main/kotlin/org/p2p/uikit/organisms/seedphrase/SeedPhraseWord.kt

```diff
@@ -2,9 +2,10 @@ package org.p2p.uikit.organisms.seedphrase
 
 data class SeedPhraseWord(
     val text: String,
-    val isValid: Boolean
+    val isValid: Boolean,
+    var isBlurred: Boolean = false
```

Comment:
it's domain entity, not a view one. i don't think it's okay to put some blur flag here

Reply:
so isValid is not OK too ))